tests/unit_tests/gui/ertwidgets/test_plot_case_selection_widget.py

```diff
@@ -2,57 +2,49 @@
 from qtpy.QtCore import Qt
 
 from ert.gui.tools.plot.plot_ensemble_selection_widget import (
-    EnsembleSelectCheckButton,
     EnsembleSelectionWidget,
+    EnsembleSelectListWidget,
 )
 
-from ..conftest import get_children
+from ..conftest import get_child
 
 
-def test_ensemble_selection_widget_maximum_selection(qtbot: QtBot):
+def test_ensemble_selection_widget_max_min_selection(qtbot: QtBot):
     test_ensemble_names = [f"case{i}" for i in range(10)]
     widget = EnsembleSelectionWidget(ensemble_names=test_ensemble_names)
     qtbot.addWidget(widget)
-    buttons = get_children(widget, EnsembleSelectCheckButton, "ensemble_selector")
+    list_widget = get_child(widget, EnsembleSelectListWidget, "ensemble_selector")
 
-    # click all buttons
-    for button in buttons:
-        qtbot.mouseClick(button, Qt.LeftButton)
-
-    maximum_selected = widget.MAXIMUM_SELECTED
     assert (
-        sorted(widget.getPlotEnsembleNames())
-        == sorted(test_ensemble_names)[:maximum_selected]
-    )
-
-
-def test_ensemble_selection_widget_minimum_selection(qtbot: QtBot):
-    test_ensemble_names = [f"case{i}" for i in range(10)]
-    widget = EnsembleSelectionWidget(ensemble_names=test_ensemble_names)
-    qtbot.addWidget(widget)
-    buttons = get_children(widget, EnsembleSelectCheckButton, "ensemble_selector")
-    for button in buttons:
-        qtbot.mouseClick(button, Qt.LeftButton)
+        len(widget.getPlotEnsembleNames()) == list_widget.MINIMUM_SELECTED
+    )  # initially one selected
 
-    checked_buttons = [button for button in buttons if button.isChecked()]
-    assert len(checked_buttons) > 0
-    for button in checked_buttons:
-        qtbot.mouseClick(button, Qt.LeftButton)
-    checked_buttons = [button for button in buttons if button.isChecked()]
-    assert len(checked_buttons) == widget.MINIMUM_SELECTED
+    qtbot.mouseClick(
+        list_widget.viewport(),
+        Qt.LeftButton,
+        pos=list_widget.visualItemRect(list_widget.item(0)).center(),
+    )  # deselect the only item selected
 
-
-def test_ensemble_selection_widget_cannot_deselect_only_active_initial_case(
```

Comment:
Is this test case now covered by the altered test?

Reply:
The 5 limit was imposed back again in 2016, and there has been no strong indication that this is a set limit other than the ui-components used and that you could not add more comboboxes when reaching the maximum set.
Ref. e713795f68fc8f08284d1f8091345334a779ce54

I agree we can remove the duplicated test.